harness/determined/cli/workspace.py

```diff
@@ -505,17 +547,32 @@ def yaml_file_arg(val: str) -> Any:
                         "set workspace-namespace binding",
                         [
                             cli.Arg("workspace_name", type=str, help="name of the workspace"),
-                            cli.Arg(
-                                "--namespace",
-                                type=str,
-                                help="existing namespace to which the workspace is bound.",
-                            ),
                             cli.Arg(
                                 "--cluster-name",
                                 type=str,
                                 help="cluster within which we create \
                                 the workspace-namespace binding",
```

Comment:
If `det w bindings myws set --namespace myns` is a valid command for user with single cluster, can you update the helper message like the one in `det w create`.